src/service/bind-to.js

```diff
@@ -5,9 +6,14 @@
  *
  */
 angular.module('ngTasty.service.bindTo', [])
-.factory('bindTo', function($parse) {
+.factory('bindTo', ["$parse", function($parse) {
```

Comment:
We use ngAnnotate to compile the ngTasty, we don't nee write manually all the annotation.